rust/hyperlane-base/src/settings/base.rs

```diff
@@ -249,7 +249,7 @@ impl Settings {
     /// Try to get the domain for a given chain by name.
     pub fn lookup_domain(&self, chain_name: &str) -> eyre::Result<HyperlaneDomain> {
         self.chains
-            .get(chain_name)
+            .get(&chain_name.to_ascii_lowercase())
```

Comment:
Where is this used? I'm surprised this is necessary

Reply:
Might be easiest to answer with a search result
https://github.com/search?q=repo%3Ahyperlane-xyz%2Fhyperlane-monorepo%20lookup_domain&type=code

This change is necessary to fix the ticket.

Reply:
That would make more sense. Changed.